src/modules/GroupChannel/context/GroupChannelProvider.tsx

```diff
@@ -335,7 +335,7 @@ const GroupChannelProvider = (props: GroupChannelContextProps) => {
     setAnimatedMessageId(0);
     setIsScrollBottomReached(true);
 
-    if (messageDataSource.hasNext()) {
+    if (config.isOnline && messageDataSource.hasNext()) {
```

Comment:
When the collection is initialized in offline mode, the initial value of **hasNext** is always true.
This leads to an issue of initializing the collection during the process of scrolling to the bottom after sending a message. To prevent this, I've added a check for **isOnline**